lib/email.js

```diff
@@ -179,42 +181,38 @@ Email.prototype.headers = function () {
  * @api Public
  **/
 Email.prototype.send = function send (callback) {
-	var self = this;
-
-  var invalid = self.validate();
+  const invalid = this.validate();
 
   if (invalid) {
     return callback(new Error(invalid));
   }
 
   // Prepare the data and send to it AWS SES REST API
 
-  var data = querystring.stringify(self.data());
-  var parsedUrl = parse(self.amazon);
-  var headers = self.headers();
+  var data = querystring.stringify(this.data());
+  var parsedUrl = parse(this.amazon);
+  var headers = this.headers();
 
   var options = {
-      uri: self.amazon
+      uri: this.amazon
     , host: parsedUrl.hostname
     , headers: headers
     , body: data
     , service: 'ses'
     , json: true
```

Comment:
👆 This line needs to be deleted, than everything will work again

Reply:
Also, maybe refactor this away from comma-first style to be consistent with your other refactoring changes :)

Reply:
Yeah, I'm not a fan of comma-first either, but I'll leave that to a future `eslint` rule to enforce.